src/app/screens/Home/AllowanceView/index.tsx

```diff
@@ -33,52 +34,44 @@ const AllowanceView: FC<Props> = (props) => {
     getFormattedNumber,
   } = useSettings();
 
-  const [payments, setPayments] = useState<Transaction[] | null>(null);
-  const [isLoadingPayments, setIsLoadingPayments] = useState(true);
+  const [transactions, setTransactions] = useState<Transaction[] | null>(null);
+  const [isLoadingTransactions, setIsLoadingTransactions] = useState(true);
 
   const { t } = useTranslation("translation", { keyPrefix: "home" });
 
   const showFiat = !isLoadingSettings && settings.showFiat;
-  const hasPayments = !isLoadingPayments && !!payments?.length;
-  const isEmptyPayments = !isLoadingPayments && payments?.length === 0;
+  const hasPayments = !isLoadingTransactions && !!transactions?.length;
```

Comment:
this could be renamed to hasTransactions

Reply:
Good catch.